devtools-ci-index.yaml

```
@@ -3777,6 +3874,22 @@
             ee_test_start_time: '20 6,18 * * *'
             timeout: 60m
             zabbix_enabled: false
+        - 'devtools-prebooster-test-{test_url}-{booster_name}':
+            test_url: openshift.io
+            prebooster_test_start_time: '5 0-23/4 * * *'
+            os_creds_username: 'username_1'
+            os_creds_password: 'password_1'
+            project_name: osio-ci-boost1-stage
```

```suggestion
            project_name: osio-ci-preboost1-stage
```
The user from vault is `osio-ci-preboost1` :)